test/views/pages/result/ResultPrintViewAmendmentSpec.scala

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.pages.result
+
+import models.PrintDisplayModel
+import org.jsoup.Jsoup
+import play.api.i18n.Messages
+import testHelpers.ViewSpecHelpers.CommonViewSpecHelper
+import testHelpers.ViewSpecHelpers.result.ResultPrintViewAmendment
+import views.html.pages.result.resultPrintViewAmendment
+
+class ResultPrintViewAmendmentSpec extends CommonViewSpecHelper with ResultPrintViewAmendment {
+
+  "The Print Result Page" should {
```

Comment:
Similarly to `OutcomeAmendedViewSpec`, there is a lot of conditions in `resultPrintView` template, but only 2 test scenarios are present. I suggest to create a test scenario for each of the 14 notification IDs.

What is more, test cases in this spec are not structured very well - test cases for all 3 models are on the same level.